src/lib/hooks/useBidProvider.tsx

```diff
@@ -202,11 +203,16 @@ export const useBidProvider = (): UseBidProviderReturn => {
 	const placeBid = useCallback(
 		async (domain: Domain, bid: number, onStep: (status: string) => void) => {
 			if (sdk?.zauction === undefined) {
-				console.warn('No zAuctionInstance');
+				console.warn(ERRORS.FAILED_TO_CHECK_ZAUCTION);
 				return;
 			}
 			if (!library) {
-				console.error('Could not find web3 library');
+				console.error(ERRORS.LIBRARY_NOT_FOUND);
+				return;
+			}
+
+			if (!domain.id) {
+				console.error(ERRORS.FAILED_TO_LOAD_DOMAIN_ID);
 				return;
 			}
```

Comment:
Could be worth `throw`ing these errors to the parent `catch` - this works as well though.